src/main/java/pro/sky/mynewproject28/controller/EmployeeController.java

```diff
@@ -0,0 +1,48 @@
+package pro.sky.mynewproject28.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.mynewproject28.model.Employee;
+import pro.sky.mynewproject28.service.impl.EmployeeServiceImpl;
+
+import java.util.Collection;
+
+@RequestMapping("/employee")
+@RestController
+public class EmployeeController {
+
+    private final EmployeeServiceImpl employeeServiceImpl;
```

Comment:
```suggestion
    private final EmployeeService employeeServiceImpl;
```
пользуемся полиморфизмом